### PR TITLE
Move selected event to ModelManager

### PR DESCRIPTION
**Pull request by theJrLinguist**

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -129,4 +130,13 @@
      */
     Person getPerson(Index targetIndex);
 
+    /**
+     * Sets the selected event.
+     */
```
Comment (yao-feng) [inline src/main/java/seedu/address/model/Model.java]: would be good to add @param tag in the javadoc comment

File: src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java [hunk 2]
```diff
@@ -33,7 +33,7 @@ public void execute_acceptedAddPoll() {
         //eventBuilder.withOrganiser(user);
         //Event event = eventBuilder.withPoll().build();
```
Comment (yao-feng) [inline src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java]: remove the unnecessary comments?

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (yao-feng) [inline src/main/java/seedu/address/model/Model.java]: would be good to add @param tag in the javadoc comment.

File: src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java (hunk 2 above)
Comment (yao-feng) [reply src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java]: would be good to add a test for the NoEventSelectedException in AddPollOptionCommand.java as shown in coveralls as well.

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (theJrLinguist) [reply src/main/java/seedu/address/model/Model.java]: Actually not sure if this is strictly required? If so we have to mass add params for every method...

File: src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java (hunk 2 above)
Comment (theJrLinguist) [reply src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java]: Ok wrt the commented out code. But the test for no event selected is there alr no?

Comment (theJrLinguist) [reply src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java]: Ok I realised the error. Will correct in next PR

**Pull request merged**